pkgs/applications/networking/cluster/minikube/default.nix

```diff
@@ -0,0 +1,42 @@
+{ stdenv, fetchurl, kubernetes }:
+let
+  arch = if stdenv.isLinux
+         then "linux-amd64"
+         else "darwin-amd64";
+  checksum = if stdenv.isLinux
+             then "17r8w4lvj7fhh7qppi9z5i2fpqqry4s61zjr9zmsbybc5flnsw2j"
+             else "0jf0kd1mm35qcf0ydr5yyzfq6qi8ifxchvpjsydb1gm1kikp5g3p";
+in
+stdenv.mkDerivation rec {
+  pname = "minikube";
+  version = "0.13.1";
+  name = "${pname}-${version}";
+
+  src = fetchurl {
+    url = "https://storage.googleapis.com/minikube/releases/v${version}/minikube-${arch}";
```

Comment:
oh, the other pull request from @offlinehacker  builds from source, so I would actually prefer it in this case.